parquet-column/src/main/java/org/apache/parquet/example/data/simple/SimpleGroup.java

```diff
@@ -48,28 +48,30 @@ public String toString() {
   }
 
   public String toString(String indent) {
-    String result = "";
+    StringBuilder result = new StringBuilder();
     int i = 0;
     for (Type field : schema.getFields()) {
       String name = field.getName();
       List<Object> values = data[i];
       ++i;
-      if (values != null) {
-        if (values.size() > 0) {
+      if (values != null && !values.isEmpty()) {
           for (Object value : values) {
-            result += indent + name;
+            result.append(indent);
+            result.append(name);
             if (value == null) {
-              result += ": NULL\n";
+              result.append(": NULL\n");
             } else if (value instanceof Group) {
-              result += "\n" + ((SimpleGroup)value).toString(indent+"  ");
+              result.append("\n");
```

Comment:
nit: Appending a char instead of a string with one char would perform a little bit better
```suggestion
              result.append('\n');
```